python/flatbuffers/compat.py

```diff
@@ -11,13 +11,16 @@
 
 if PY3:
     string_types = (str,)
-    binary_type = bytes
+    binary_types = (bytes,bytearray)
     range_func = range
     memoryview_type = memoryview
     struct_bool_decl = "?"
 else:
-    string_types = (basestring,)
-    binary_type = str
+    string_types = (unicode,)
+    if PY26 or PY27:
+        binary_types = (str,bytearray)
```

Comment:
The thing I was thinking about was `buffer` vs `memoryview` (https://github.com/google/flatbuffers/blob/master/python/flatbuffers/compat.py) and you were correct. Looks good!